### DISCUSSION TITLE
Trigger workflow in other repo using workflow_dispatch and `GITHUB_TOKEN`

### DISCUSSION THREAD

**Discussion by michaelweinold** [community/community#42316] [Actions] [answered]
### Select Topic Area

Question

### Body

I am trying to trigger a workflow in one repository (1) by running a workflow in another repository (2).

Both repositories are within the same organization. To avoid having to re-issue personal access tokens (PAT), I would like to do this using just the `GITHUB_TOKEN`. 

[This blog article from September 2022](https://github.blog/changelog/2022-09-08-github-actions-use-github_token-with-workflow_dispatch-and-repository_dispatch/) mentions that triggering workflows from a workflow this is now possible with just the `GITHUB_TOKEN`.

Does this work between different repositories also?

In repository (2), I have:

```
name: Create Workflow Dispatch

on:
  push:
    branches:
      - main  
  workflow_dispatch:

jobs:
  build:
    permissions: write-all
    runs-on: ubuntu-latest
    steps:
      - name: Trigger Workflow
        uses: actions/github-script@v6
        with:
          script: |
            github.rest.actions.createWorkflowDispatch({
              owner: 'org_name',
              repo: 'repo_name',
              workflow_id: 'action.yml',
              ref: 'main',
            })
```

but this gives me the following error message: `Error: Unhandled error: HttpError: Resource not accessible by integration`.

**Comment by michaelweinold** [comment]
Compare also: https://github.com/orgs/community/discussions/26323#discussioncomment-4217946

**Comment by airtower-luna** [comment, marked as answer]
> Does this work between different repositories also?

No, the [documentation on the `GITHUB_TOKEN`](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret) states:

> The token's permissions are limited to the repository that contains your workflow.
